ngx-http-concat.php

```diff
@@ -94,8 +94,14 @@ function concat_get_path( $uri ) {
 // /foo/bar.css,/foo1/bar/baz.css?m=293847g
 // or
 // -eJzTT8vP109KLNJLLi7W0QdyDEE8IK4CiVjn2hpZGluYmKcDABRMDPM=
-if ( '-' == $args[0] )
-	$args = gzuncompress( base64_decode( substr( $args, 1 ) ) );
+if ( '-' == $args[0] ) {
+	$args = @gzuncompress( base64_decode( substr( $args, 1 ) ) );
+
+	// Invalid data, abort!
+	if ( false === $args ) {
+		die();
```

Comment:
Instead of the `die()`, this should use `concat_http_status_exit( 400 )` so that proper headers are sent. Among other things, those headers will stop the empty response from being cached/served.